magento/pwa-studio
# #3038 Use @apollo/client in "Modify talon results" tutorial

## Description

Replaces the reference to `@apollo/react-hooks` with `@apollo/client` in line with other packages in PWA studio v8.0.0.

## Related Issue





Closes #3038.

### Verification Steps


1. Follow this tutorial https://magento.github.io/pwa-studio/tutorials/intercept-a-target/modify-talon-results/
2. Add the created extension to your main PWA studio project
3. Ensure the site compiles and behaves as expected.
